src/App.jsx

```diff
@@ -1,104 +1,32 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList/PostList';
+
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+function getCommetnsById(postId) {
+  return (
+    commentsFromServer.filter(comment => {
+      return comment.postId === postId;
+    }) || null
```

Comment:
The `|| null` part is unnecessary here because `filter` always returns an array. If no comments match, it will return an empty array, which is usually preferable to `null`.